cob_description/urdf/sensors/prosilica.gazebo.xacro

```diff
@@ -25,12 +25,12 @@
         </camera>
         <plugin name="${name}_controller" filename="libgazebo_ros_camera.so">
           <updateRate>10.0</updateRate>
-          <cameraName>${name}_sensor</cameraName>
-          <frameName>/${name}_link</frameName>
+          <cameraName>${name}</cameraName>
+          <frameName>${name}_link</frameName>
           <topicName>${ros_topic}_image_data</topicName>
-          <imageTopicName>/${ros_topic}/image_raw</imageTopicName>
-          <cameraInfoTopicName>/${ros_topic}/camera_info</cameraInfoTopicName>
-          <pollServiceName>/${name}/request_image</pollServiceName>
+          <imageTopicName>${ros_topic}_image_raw</imageTopicName>
+          <cameraInfoTopicName>${ros_topic}_camera_info</cameraInfoTopicName>
```

Comment:
So what fully qualified topic_name does this lead to?
Is it `cameraName/imageTopicName`? Then I'd suggest to get rid of the `ros_topic` property and just set `<imageTopicName>image_raw</imageTopicName>`, so that it leads us to something like `/robot_1/head_color_camera_l/image_raw`